spec/javascripts/health-care/health-care.spec.js

```diff
@@ -201,5 +211,31 @@ describe("Health Care Form", function() {
         expect(true).to.be.false;
       }).always(function() { done(); });
   });
+
+  it("should return true for valid monetary value input", function() {
+    var HealthApp = document.getElementById("fixture-frame").contentWindow.HealthApp;
+    var form = HealthApp.getFormRoot();
+    var testElement = form.elements['veteran[gross_wage_income]'];
+    var validInput = ['55000.00', '100', '1.99'];
+
+    for (var i = 0; i < validInput.length; i++) {
+      testElement.value = validInput[i];
+      var valid = HealthApp.validate(testElement);
```

Comment:
expect(HealthApp.validate(testElement).to.be.true; will yield a better error message on failure because it will show the HealthApp.validate() call in the error string.